content/blog/blog-2020-07-01-kudo-dependencies.md

```diff
@@ -0,0 +1,93 @@
+---
+date: 2020-07-01
+---
+
+# KUDO Dependencies
+
+One of the most requested features (internally and externally) has been operator dependencies. We've experienced this pain ourselves: our own [Kafka](https://github.com/kudobuilder/operators/tree/master/repository/kafka) operator requires the user to manually [install Zookeeper](https://github.com/kudobuilder/operators/blob/master/repository/kafka/docs/v1.3/install.md) as a prerequisite. This sounds simple enough but can get out of hand quickly as our [Flink demo](https://github.com/kudobuilder/operators/tree/master/repository/flink/docs/demo/financial-fraud) shows.
+
+We've been thinking about the best way of introducing dependencies to KUDO for a while now. A few months ago we created [KEP-29](https://github.com/kudobuilder/kudo/blob/main/keps/0029-operator-dependencies.md) that aimed at solving at least part of the problems. Dependencies can be a complex topic. KEP-29 is not trying to boil the dependency ocean but rather limits itself to installation dependencies only, i.e. your operator instance and all its dependencies (including transitive ones) will be installed and/or removed as one unit.
```

Comment:
"at least part of the problems" -> I would lean towards either "part of the problem" or "part of the set of problems".

Reply:
I think I would be more direct with " Dependencies can be a complex topic" -> "Dependencies is a complex topic"

Reply:
> "at least part of the problems" -> I would lean towards either "part of the problem" or "part of the set of problems".

"part of the problem" sounds better to my ears.

Reply:
> I think I would be more direct with " Dependencies can be a complex topic" -> "Dependencies is a complex topic"

Maybe "dependency handling is a complex topic"? "Dependencies is a complex topic" sounds weird to my non-native ears, I want to make it "Dependencies are a complex topic"...